src/app/services/scheduler/scheduler.service.ts

```diff
@@ -87,8 +87,19 @@ export class SchedulerService {
             if (allocations && allocations.length > 0) {
               const appAllocations = [];
               allocations.forEach(alloc => {
+                if (
+                  alloc.allocationTags['kubernetes.io/meta/namespace'] &&
```

Comment:
Change this if check to `alloc.allocationTags && alloc.allocationTags['kubernetes.io/meta/namespace'] && alloc.allocationTags['kubernetes.io/meta/podName']`